src/main/java/org/mapdb/Serializer.java

```diff
@@ -25,108 +24,124 @@
 import java.util.*;
 
 /**
- * Provides serialization and deserialization
+ * This interface specifies how Java Objects are serialized and de-serialized
+ * and also how objects are compared, hashed and tested for equality for use
+ * with MapDB.
+ * <p>
```

Comment:
It is standard now to use the single <p> tag when starting a new paragraph. Check out java.util.Stream for example.